apps/masterbots.ai/lib/hooks/use-profile.tsx

```
@@ -1,20 +1,22 @@
 'use client'
 
 import { getUserBySlug, updateUserPersonality } from '@/services/hasura'
-import type { User } from 'mb-genql'
+import { User } from 'mb-genql'
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)